libretroshare/src/ft/ftserver.cc

```diff
@@ -1356,7 +1357,7 @@ bool ftServer::decryptItem(const RsTurtleGenericDataItem *encrypted_item,const R
 	decrypted_item = dynamic_cast<RsTurtleGenericTunnelItem*>(deserialise(data,&data_size)) ;
 	free(data);
 
-	return (decrypted_item != NULL);
```

Comment:
This is the limit to what I can accept: the test returns a bool as an explicit calculation. If you return the object itself, it is implicitely converted to a bool, which is much less clear and prone to wrong interpretation (not to mention the syntax with () that are not needed). So I would definitely keep the if(decrypted_item != NULL) here.

Reply:
I'm OK with that.